beetsplug/replaygain.py

```diff
@@ -252,6 +260,8 @@ def compute_chunk_gain(self, items, is_album, target_level):
         cmd = [self.command]
         cmd += ["--" + method]
         cmd += ['--xml', '-p']
+        if self.version >= '0.6.0':
```

Comment:
Good point, it's not...
If we want to be really sure we could also go
``` python
$ from packaging import version
$ version.parse('0.10.0c') >= version.parse('0.6.0b')
True
```

Reply:
As nifty as this is, I would love to avoid using `packaging` because it is technically a new dependency—it is [not in the standard library](https://docs.python.org/3/library/packaging.html).